src/fh-shared-kernel.shared-kernel/Reports/ConnectionRequests/ConnectionRequests.cs

```diff
@@ -0,0 +1,12 @@
+namespace FamilyHubs.SharedKernel.Reports.ConnectionRequests;
+
+public class ConnectionRequests
+{
+    public string? Date { get; set; }
```

Comment:
Are you planning on returning the `ConnectionRequests` object for total / past 7 days queries as well and just omitting the Date?

Would it make more sense to remove it from here and have another class that extends this and adds Date?

Reply:
Yeah that's my plan, here's what it would look like from a JSON perspective:

```
{
  "totals": {
    "date": null,
    "made": 0,
    "accepted": 0,
    "declined": 0
  },
  "weeklyReports": [
    {
      "date": "3 June to 9 June",
      "made": 0,
      "accepted": 0,
      "declined": 0
    },
    {
      "date": "10 June to 16 June",
      "made": 0,
      "accepted": 0,
      "declined": 0
    },
    {
      "date": "17 June to 23 June",
      "made": 0,
      "accepted": 0,
      "declined": 0
    },
    {
      "date": "24 June to 30 June",
      "made": 0,
      "accepted": 0,
      "declined": 0
    }
  ]
}
```

I'm happy with either my original approach or your suggestion, what would you prefer to deal with on the UI side?

Reply:
I _think_ it would make more sense to separate them... Probably make it slightly easier to work with.

Reply:
Sounds good, updated in [90eda07](https://github.com/DFE-Digital/fh-shared-kernel/pull/70/commits/90eda07553326149698f021eb918119a4be26b6a)